diff --git a/node_modules/tempa-xlsx/xlsx.js b/node_modules/tempa-xlsx/xlsx.js
index 6bca3c6..4a999af 100644
--- a/node_modules/tempa-xlsx/xlsx.js
+++ b/node_modules/tempa-xlsx/xlsx.js
@@ -12151,7 +12151,7 @@ var XmlNode = (function () {
     return this;
   }
 
-  var APOS = "'"; QUOTE = '"'
+  var APOS = "'", QUOTE = '"'
   var ESCAPED_QUOTE = {  }
   ESCAPED_QUOTE[QUOTE] = '&quot;'
   ESCAPED_QUOTE[APOS] = '&apos;'
